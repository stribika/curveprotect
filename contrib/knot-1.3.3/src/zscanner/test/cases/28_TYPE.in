$ORIGIN	.
$TTL	1

; OK
@	TYPE0		\# 0			; Minimal type number
@	TYPE65535	\# 0			; Maximal type number
@	TYPE55555	\# 0			; Without hex rdata
@	TYPE55555	\# 1 00			; Without hex rdata
@	TYPE1		\# 4 00000000		; Known type
@	TYPE1		0.0.0.0			; Known type in text format
@	TYPE55555	(			; Multiline begin
	\#
	5
	0102 03
	04 05
	)					; Multiline end
@	type55555	\# 0			; Type in lower-case

; KO
@	TYPE55555
@	TYPE55555				; Without text rdata
@	TYPE65536				; Type number overflow
@	TYPE65535x				; Bad type
@	TYPE55555	\#			; Missing hex length
@	TYPE55555	\# 1 0000		; Too long rdata
@	TYPE55555	\# 2 00			; Bad rdata length
@	TYPE55555	\# 1 00 x		; Unexpected data
