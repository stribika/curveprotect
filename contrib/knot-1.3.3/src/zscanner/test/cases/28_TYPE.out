OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0000
RDATA=
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=FFFF
RDATA=
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=D903
RDATA=
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=D903
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0001
RDATA=00000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0001
RDATA=00000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=D903
RDATA=0102030405
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=D903
RDATA=
------
WARNG=ZSCANNER_ECANNOT_TEXT_DATA
------
WARNG=ZSCANNER_ECANNOT_TEXT_DATA
------
WARNG=ZSCANNER_ENUMBER16_OVERFLOW
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_RDATA_LENGTH
------
WARNG=ZSCANNER_EBAD_RDATA_LENGTH
------
WARNG=ZSCANNER_EBAD_HEX_CHAR
------
